src/ui/public/notify/notifier.js

```diff
@@ -310,48 +320,69 @@ Notifier.prototype._showFatal = function (err) {
  * @param  {Error|String} err
  * @param  {Function} cb
  */
-Notifier.prototype.error = function (err, cb) {
-  return add({
+Notifier.prototype.error = function (err, opts, cb) {
+  const overrideable = ['lifetime', 'icon'];
+  if (_.isFunction(opts)) {
+    cb = opts;
+    opts = {};
+  }
```

Comment:
I left comments on a PR that was merged into this one about these conditions. This is helping to get the blocker issue fixed by not breaking an API, but I dug around and can't really see anywhere that `error` `info` and `warning` notifications actually use the callback param anywhere.